### ISSUE TITLE
WCAG: Labels read twice by screen reader

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#227]
### Description of the bug

In an external Screen reader test it was reported that the labels were read twice. 
Could not recreate this with VoiceOver/Chrome. Should be checked on more screen readers / browsers

### Steps To Reproduce

Use a screen reader to go trough a form

### Additional Information

This was tested on Oppstartsveileder
Need to find out what screen reader and browser was used in the test

**Comment by Febakke** [comment]
Not been able to recreate this bug.
